src/compiler/compile/utils/is_interactive_role.ts

```diff
@@ -0,0 +1,18 @@
+import { ARIARoleDefintionKey, roles as rolesMap } from 'aria-query';
+
+const roles = [...rolesMap.keys()];
+const _interactiveRoles = roles
+	.filter((name) => !rolesMap.get(name).abstract)
+	.filter((name) =>
+		rolesMap.get(name).superClass.some((c) => c.includes('widget'))
+	);
```

Comment:
Couldn't these be combined into a single filter? I believe the second condition should get short circuited if the former evaluates to false.

Reply:
you’re right 👍  I fixed in 39316a4